portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,88 @@
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<head>
+  <title>Jason Han</title>
+
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <!-- Local imports  -->
+  <link rel="stylesheet" type="text/css" href="css/index.css" />
+  <link rel="stylesheet" type="text/css" href="css/global.css" />
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script type="text/javascript" src="nav.js"></script>
+
+  <!-- Fonts -->
+  <link
+    href="https://fonts.googleapis.com/css?family=Quicksand|Josefin+Sans:300|Raleway"
+    rel="stylesheet"
+  />
+  <script
+    defer
+    src="https://use.fontawesome.com/releases/v5.0.13/js/all.js"
+    integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe"
+    crossorigin="anonymous"
+  ></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="navbar-wrapper">
+      <span class="navbar-toggle" id="js-navbar-toggle"></span>
+      <a href="index.html" class="logo"><b>Jason Han</b></a>
+      <ul class="main-nav" id="js-menu">
+        <li>
+          <a href="index.html" class="nav-links">About</a>
+        </li>
+        <li>
+          <a href="projects.html" class="nav-links">Projects</a>
+        </li>
+        <li>
+          <a href="resume.html" class="nav-links">Resume</a>
+        </li>
+        <li>
+          <a href="blog.html" class="nav-links">Blog</a>
+        </li>
+      </ul>
     </div>
-  </body>
-</html>
+  </nav>
+  <div id="wrapper">
+    <div class="main-content">
+      <img class="me-vert" src="images/portrait.jpg" />
+      <img class="me-horiz" src="images/portrait.jpg" />
+      <div class="main-content-text">
+        <h2 class="hello-text">Hello,</h2>
+        <p>
+          My name is <b>Jason Han</b>. I'm from Jersey City, New Jersey, and I'm
+          a prospective Computer Science / Economics major at Vassar College! My
+          goal is to explore as much as I can in my freshman year and focus on
+          my interests – CS, economics, physics, and tennis, to name a few.
+        </p>
+        <p>
+          Feel free to contact me at
+          <a href="mailto:jjhan@vassar.edu">jjhan@vassar.edu</a> or through my
+          Instagram (at the bottom). Feel free to also check out my LinkedIn or
+          my GitHub.
+        </p>
+      </div>
+    </div>
+
+    <footer class="footer">
+      <img class="divider" src="images/divider.png" />
+      <div class="links">
+        <span><a href="mailto:jjhan@vassar.edu">Email</a></span>
+        <span>/</span>
+        <span
+          ><a href="https://www.linkedin.com/in/jasonhan-vassar/"
+            >Linkedin</a
+          ></span
+        >
```

Comment:
did prettier put these `>` on the next line?

Reply:
it actually did, I just tried reproducing! I'll double check after formatting from now on